doc/source/examples.rst

```diff
@@ -1,9 +1,12 @@
 Examples
```

Comment:
Change to `Usage`

Reply:
https://github.com/ComPWA/pycompwa/pull/74/commits/3f934c0e4826571b8bf9bd64462b7db83f128636